stdlib/public/stubs/Stubs.cpp

```diff
@@ -140,12 +140,21 @@ static int swift_snprintf_l(char *Str, size_t StrSize, locale_t Locale,
 #endif
 
 template <typename T>
+static int swift_floatingPointToStringPrecision(bool Debug) {
```

Comment:
It's not totally obvious to me that this warrants a separate function.  Do we expect this to get more logic to become more complex in the future?  If not, it seems clearer to me if this is just folded into the caller.